src/ViewModels/ArcGisMapServerItemViewModel.js

```diff
@@ -43,7 +43,7 @@ defineProperties(ArcGisMapServerItemViewModel.prototype, {
      */
     type : {
         get : function() {
-            return 'esri-mapServer';
+            return 'esri-rest';
```

Comment:
This is in line with ckan definitions from what I could find and most esri sites.